account_bank_statement_import_camt_details/models/account_bank_statement_line.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+"""Add process_camt method to account.bank.statement.import."""
+# © 2017 Compassion CH <http://www.compassion.ch>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import models, fields
+
+
+class AccountBankStatementLine(models.TransientModel):
```

Comment:
Oops, this should be `models.Model` instead of `models.TransientModel`.  It will break in 10.0. ([evidence](https://travis-ci.org/lbeopennet/ci-staging/jobs/236312784#L712))